src/chainstate/stacks/miner.rs

```diff
@@ -1010,7 +1010,7 @@ impl<'a> StacksMicroblockBuilder<'a> {
         let deadline = get_epoch_time_ms() + (self.settings.max_miner_time_ms as u128);
         let mut block_limit_hit = BlockLimitFunction::NO_LIMIT_HIT;
 
-        mem_pool.reset_last_known_nonces()?;
```

Comment:
We're not using any of the methods that manipulate the last_nonces columns, right?  If so, then can you delete those as well, and add a comment to the schema that added these columns that they are no longer used?

Reply:
Sure. Just to be clear, you want to remove those columns from the DB in the latest schema in addition to deleting all of the related code?

Reply:
If it's not too much trouble -- i.e. if it can be done with a `DROP COLUMN`.  Sqlite has a bunch of constraints on when you can and cannot do this, however, so don't worry about it if sqlite is preventing you.

Reply:
I'm seeing:

```
thread 'main' panicked at 'Failed to open mempool db: SqliteError(SqliteFailure(Error { code: Unknown, extended_code: 1 }, Some("near \"DROP\": syntax error")))', src/main.rs:739:14
```

But strangely, I can run the same query from the command line and it works.

```
ALTER TABLE mempool DROP COLUMN last_known_origin_nonce;
```

Reply:
Maybe sqlite allows it but `rusqlite` does not?